meetings/2023-01-25.md

```diff
@@ -46,14 +56,67 @@
+* SC: I will demo this properly some other time, but I want to show this: in the notifications panel, our concepts are very well defined, and one of the things required by ??? is that a specification define classes of products. For example in notifications we define what interoperability entails, and so on. These concepts are accessible. This application extracts all the concepts available, but also all concepts that are included in all cited documents: all dependencies or references that this article has, we can fetch some documents and build an additional set of concepts. All of this is still available from any document. And you can see a graph of those concepts. 
+* SC: The TR links to all these work items. The idea was to build a Solid glossary from all the specs. 
+* PAC: Thanks for the demo. All those sections giving definitions from other specifications, they are pulled automatically from the ??? specification? 
+* SC: Yes.
+* PAC: That document with the definitions, is it one given spec or a central glossary? 
+* SC: This is the same as (URL). 
+* PAC: IT grabs concepts from RDF descriptions?
+* SC: Yes. If resource is returning RDFa/Turtle/JSON-LD it is all the same. 
+* SC: All our technical reports can potentially provide this information. 
+* eP: Since you mentioned content negotiation, can solidproject.org support that? 
+* SC: https://github.com/solid/solidproject.org/issues/248 :) 
+* SC: But I don't think we can do that with GitHub. Right now we have HTML served from these URIs of these documents, some of them have RDFa.
+* eP: If you had it in a Solid server I don't know if implementations support that HTML ??? Does CSS support RDFa to turtle? 
+* SC: There is no requirement. 
+* SC: I know there are some issues to improve how we describe containers, and where RDFa fits in with writing/patching. 
+* eP: So it would rely on specific implementation and not protocol level. 
+* SC: Yes. Some implementations are doing that, equipped with multiple RDF parsers, but on a spec level the minimum requirement is that turtle and json-ld are available for RDF resources. NSS does it, CSS does it, I don't know about others. I think there may be some interest in servers converting ??? 
+* SC: Notification protocol is a good example of how to do conformance properly, really specifying the products that are expected to be implemented for it. And the communication between two products. 
+* PAC: That is interesting. I need to dig more as what spec is doing when dereferencing definitions in other specs, because we have a lot of specifications that are not using RDF, but the semantics of HTML could be leveraged. Not so much about tests and requirements, but concepts and definitions. 
+* SC: There is webref (URL). You may be able to link to a requirement as HTML but there is no way of expanding a requirements ??? We're doing that because maybe it's useful but also demonstrates Solid. 
```

Comment:
```suggestion
* SC: There is webref (URL). You may be able to link to a requirement as HTML, but there is no way of expanding a requirements ??? We're doing that because maybe it's useful but also demonstrates Solid. 
```